app/helpers/textual_listview.rb

```diff
@@ -11,6 +11,7 @@ def locals
   end
 
   def self.new_from_hash(h)
+    h ||= {:title => "Variables", :headers => ["Name", "Value"], :col_order => ["name", "value"], :value => []}
```

Comment:
Is `:title => "Variables"` applicable to every textual listview?

Reply:
I think you're right, tomorrow I can try and create a more generic fix for this and revert this function back to how it was before.